tasks.py

```diff
@@ -666,9 +667,21 @@ def docs(
     )
 
     print("Running invoke docs from:", repo_root)
-    old_pwd = pathlib.Path.cwd()
+    old_cwd = pathlib.Path.cwd()
     docusaurus_dir = repo_root / "docs/docusaurus"
     os.chdir(docusaurus_dir)
+    pull_request = os.environ.get(  # noqa: TID251 # os.enrivon allowed in config files
```

Comment:
nit: `s/enrivon/environ/`

Reply:
Could you add a comment that netlify will set the env variable?

Reply:
yup, will do!